server/app.js

```diff
@@ -31,6 +33,8 @@ app.use("/ping", pingRouter);
 app.use("/api", register);
 app.use("/api", signin);
 
+/*FriendList*/
+app.use("/api/friendList", friendList);
```

Comment:
This one can maybe be plural as well
```suggestion
app.use("/api/friendLists", friendList);
```